src/wp-includes/blocks.php

```diff
@@ -753,23 +759,24 @@ function excerpt_remove_blocks( $content ) {
 }
 
 /**
- * Render inner blocks from the `core/columns` block for generating an excerpt.
+ * Render inner blocks from the allowed wrapper blocks
+ * for generating an excerpt.
  *
- * @since 5.2.0
+ * @since 5.8
  * @access private
  *
- * @param array $columns        The parsed columns block.
+ * @param array $parsed_block   The parsed block.
  * @param array $allowed_blocks The list of allowed inner blocks.
  * @return string The rendered inner blocks.
  */
-function _excerpt_render_inner_columns_blocks( $columns, $allowed_blocks ) {
+function _excerpt_render_inner_blocks( $parsed_block, $allowed_blocks ) {
```

Comment:
If renaming the function is desired, we need to move `_excerpt_render_inner_columns_blocks()` to `wp-includes/deprecated.php` and update it's code to contain a `_deprecated_function()` call, and return the result of the new function.

Even though this is marked `private`, best efforts should be made to prevent fatal errors.

Reply:
Fair point. Added in https://github.com/WordPress/wordpress-develop/pull/1472/commits/541d2033154b2b5165cedb909de40b9832419959